backend/api/grants/forms.py

```diff
@@ -1,15 +1,28 @@
-from api.forms import ContextAwareModelForm
-from conferences.models import Conference
 from django import forms
+from django.core import exceptions
+from django.utils.translation import gettext_lazy as _
 
-from .models import Grant
+from api.forms import ContextAwareModelForm
+from conferences.models import Conference
+from grants.models import Grant
 
 
 class GrantForm(ContextAwareModelForm):
     conference = forms.ModelChoiceField(
         queryset=Conference.objects.all(), to_field_name="code", required=True
     )
 
+    def clean(self):
+        cleaned_data = super().clean()
+
+        conference = cleaned_data.get("conference", None)
+
+        if not conference and self.instance:
+            conference = self.instance.conference
+
+        if not conference.is_grants_closed:
+            raise exceptions.ValidationError(_("The grants form is now closed!"))
```

Comment:
what happens in the FE when this error appears?

Reply:
Now we will show 404 page
```javascript
  if (!deadline) {
    return <ErrorPage statusCode={404} />;
  }
```